src/components/dashboard/index.js

```diff
@@ -99,47 +95,17 @@ const Dashboard = (props) => {
     const classes = useStyles();
     const { t } = useTranslation("dashboard");
     const { t: i18nPref } = useTranslation("preferences");
+    const { t: i18nIntro } = useTranslation("intro");
     const [userProfile] = useUserProfile();
 
-    const [bootstrapQueryEnabled, setBootstrapQueryEnabled] = useState(false);
-    const [bootstrapError, setBootstrapError] = useState(null);
-    const [bootstrapInfo, setBootstrapInfo] = useBootstrapInfo();
-
-    //get from cache if not fetch now.
-    const bootstrapCache = queryCache.getQueryData(BOOTSTRAP_KEY);
-
-    const preProcessData = useCallback(
-        (respData) => {
-            let pref = respData.preferences;
-            setBootstrapInfo({ ...bootstrapInfo, preferences: pref });
-        },
-        [bootstrapInfo, setBootstrapInfo]
-    );
-
-    useEffect(() => {
-        if (bootstrapCache && !bootstrapInfo) {
-            preProcessData(bootstrapCache);
-        } else {
-            setBootstrapQueryEnabled(true);
-        }
-    }, [preProcessData, bootstrapInfo, bootstrapCache]);
-
-    useBootStrap(
-        bootstrapQueryEnabled,
-        (respData) => preProcessData(respData),
-        setBootstrapError
-    );
```

Comment:
This code is not needed at all. I updated AppBar to update bootstrap context when it fetches it for the first time.